<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="waveformCombobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sinus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dreieck</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rechteck</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SÃ¤gezahn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rauschen</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="gainSlider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="note_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="note_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="note_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dialAttack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QDial" name="dialDecay">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="value">
     <number>35</number>
    </property>
   </widget>
   <widget class="QDial" name="dialSustain">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QDial" name="dialRelease">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Decay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sustain</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAttack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDecay">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSustain">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelease">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dialAttack</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelAttack</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialDecay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelDecay</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialSustain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelSustain</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialRelease</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelRelease</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
